src/containers/EmployeeList/Employee/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { DETAIL, TO_EMPLOYEES } from '@/configs';
+import { Box, Button, Skeleton } from '@mantine/core';
+import { Link } from 'react-router-dom';
+import styles from './styles.module.scss';
+
+interface EmployeeProps {
+  loading: boolean;
+  id: string;
+  name: string;
+  role: string;
+  image: string;
+}
+
+export default function Employee({
+  loading,
+  id,
+  name,
+  role,
+  image,
+}: EmployeeProps) {
+  const handleDetailClick = () => {
+    console.log('click');
+  };
+  return (
+    <div className={styles.userBoxContainer}>
+      <div className={styles.userBox}>
+        <Box
```

Comment:
@cpea2506 Can you check this for me?

Reply:
the userbox is necessary, i have changed my UI then i forgot to delete it from the lasted